### DISCUSSION TITLE
[feat] top page show cases indicator

### DISCUSSION THREAD

**Discussion by bracinho2** [SimformSolutionsPvtLtd/flutter_showcaseview#416] [Ideas]
Hi, guys!

Your package is super wonderful, however, I needed to use it in a request that had a top indicator on the page with the exact number of showcases... something similar to WhatsApp stories that maintain a story indicator at the top of the page .

I forked the project to implement this functionality and I ask: can I submit this idea for you to analyze whether or not it is worth including in the project?

thanks!

Alex
Brazil

**Comment by aditya-chavda** [comment]
Hey @bracinho2,
Thanks for the appreciation! Can you please share with us a screenshot of what you have achieved if possible?

There hasn't been much support for this request so we are leaning towards not considering, but I am curious why you had to fork the package to implement this feature.

Based on the outcome you achieved, we can think of something generic to be included 😄
